georchestra-integration/externalized-accounts/src/main/java/org/geonetwork/security/external/model/CanonicalGroupImpl.java

```diff
@@ -78,6 +80,9 @@ private CanonicalGroupImpl(Builder builder) {
         return linkage;
     }
 
+    @Override
```

Comment:
We need overrides annotation everywhere ? I think I've missed something. 

Couldn't we use Lombok to make files cleaner ?

Reply:
> Couldn't we use Lombok to make files cleaner ?

Lombok is not a dependency for this module right now, we might add it but maybe not in the scope of this PR ?

> We need overrides annotation everywhere ? I think I've missed something.

I don't know to be honest, I just imitated the already existing code. Relying on http://www.javapractices.com/topic/TopicAction.do?Id=223#:~:text=Using%20the%20%40Override%20annotation%20prevents,or%20implement%20an%20interface%20method. it looks like we need it.